webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/rendering/model/VSpan.java

```diff
@@ -40,20 +41,20 @@
     public VSpan(AnnotationLayer aLayer, AnnotationFS aFS, String aType, VRange aOffsets,
             Map<String, String> aFeatures)
     {
-        this(aLayer, new VID(getAddr(aFS)), aType, asList(aOffsets), aFeatures);
+        this(aLayer, new VID(getAddr(aFS)), aType, Collections.singletonList(aOffsets), aFeatures);
```

Comment:
I have had a look at the implementations of `asList()` and `singletonList()`. Actually, I wasn't aware that `asList()` internally simply calls `new ArrayList<>(...)`. I thought it would return a non-modifiable list. Well, always a good thing to learn. In fact, I was wishfully thinking that `asList(...)` may internally even switch between a multi-element and a single-element list implementation depending on the number of elements passed to it. I am not sure if (IntelliJ is smart enough to detect if) the switch from `asList()` to `singletonList()` safe - one is modifiable, the other is not. We could bulk-switch to`singletonList()` and fix those cases were we might get exceptions then (although I may have a hard time switching my head do that). I am a bit ambivalent... What do you think about the risk of getting exceptions due to the switch?